shuup/simple_cms/models.py

```diff
@@ -64,6 +64,7 @@ def for_shop(self, shop):
 class Page(MPTTModel, TranslatableModel):
     shop = models.ForeignKey("shuup.Shop", verbose_name=_('shop'))
     available_from = models.DateTimeField(
+        default=now,
```

Comment:
Would this work better? https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.DateField.auto_now_add

Reply:
Not really no, the problem with that is that it automatically sets the "Available from" date whether you assign it a value or not. This becomes an issue when the merchant wants the page to become available in 2 days, auto_now_add will set it to today's date.

Reply:
I guess we could just leave the field empty and set it to today's date if it's sent empty (just like we do it now).

Reply:
but will it possible to clear the field if I want?

Reply:
### Yes but the value will still be set to default by our backend. Perhaps, we should, set the form's initial value to be today's date, and if it's deleted by the merchant the available from value should be null?

Reply:
that is what I think, at least.

Reply:
@chessbr This works as we talked ^^, If default date is deleted from the field, the available from will be set to null.